src/geometry.rs

```diff
@@ -2434,22 +2428,15 @@ impl<'a> Geometry<'a> {
     /// ```
     /// use geos::{Geom, Geometry};
     ///
-    /// let geom = Geometry::new_from_wkt("POINT(100 90)").expect("Invalid geometry");
-    /// let small_geom = geom.buffer(25., 8).expect("buffer failed");
-    /// let big_geom = geom.buffer(50., 8).expect("buffer failed");
-    ///
-    /// let union_geom = small_geom.union(&big_geom).expect("union failed");
-    /// let build_area_geom = union_geom.build_area().expect("build_area failed");
+    /// let geom = Geometry::new_from_wkt("GEOMETRYCOLLECTION( \
```

Comment:
This simplifies the example to use far fewer coordinates, and looks like this:

<img width="310" alt="image" src="https://user-images.githubusercontent.com/3375604/182946701-0ba35953-5884-4a6b-8af0-b23f0a2eb790.png">